proposals/allow_policy_exception.md

```diff
@@ -0,0 +1,358 @@
+# Policy Exceptions
+
+- **Authors**: Eileen Yu (@Eileen-Yu), Jim Bugwadia (jim@nirmata.com)
+- **Created**: Sep 16th, 2022
+- **Updated**: Sep 26th, 2022
+- **Abstract**: Allow managing policy exceptions (exclude) independently of policies
+
+## Contents
+
+- [Introduction](#introduction)
+- [The Problem](#the-problem)
+- [Proposed Solution](#proposed-solution)
+  - [High-Level Flow](#high-level-flow)
+  - [Ideal Usage](#ideal-usage)
+  - [Possible User Stories](#possible-user-stories)
+- [Implementation](#implementation)
+- [Alternative Solution](#alternative-solution)
+- [Next Steps](#next-steps)
+
+## Introduction
+
+Kyverno policy rules contain an `exclude` block which allows excluding admission requests by resource (name, API group, versions, and kinds), resource labels, namespaces, namespace labels, users, use groups, service accounts, user role and groups. This construct allows flexibility in selecting when a policy should be applied to a admission review request.
+
+This proposal introduces the concept of a `PolicyException` which is similar to the `exclude` declaration but is decoupled from the policy lifecycle.
```

Comment:
Let's clearly state this proposal is only applicable to validate rule types. That is correct, yes?

Reply:
We suppose it's not only limited to `validation`. 
These exceptions would be added to the policy rule `exclude` block in Kyverno finally, so should share similar usage with current `exclude`.